lib/tutorials/en_US/auth.md

```diff
@@ -27,44 +23,43 @@ const users = {
     }
 };
 
-const validate = function (request, username, password, callback) {
+const validate = async (request, username, password) => {
+
     const user = users[username];
     if (!user) {
-        return callback(null, false);
+        return { credentials: null, isValid: false };
     }
 
-    Bcrypt.compare(password, user.password, (err, isValid) => {
-        callback(err, isValid, { id: user.id, name: user.name });
-    });
+    const isValid = await Bcrypt.compare(password, user.password);
+    const credentials = { id: user.id, name: user.name };
+
+    return { isValid, credentials };
 };
 
-server.register(Basic, (err) => {
+const start = async () => {
 
-    if (err) {
-        throw err;
-    }
+    const server = Hapi.server({ port: 4000 });
+
+    await server.register(require('hapi-auth-basic'));
+
+    server.auth.strategy('simple', 'basic', { validate });
+    server.auth.default('simple');
```

Comment:
I notice that you decided to use a default strategy rather than configure it on the route.  If that's a change we want to make, then we should consider updating the text below which reads,

> The last thing we do is tell a route to use the strategy named `simple` for authentication.

Reply:
Good point. I've changed it to put it back on `options`.